### ISSUE TITLE
Series Helper example giving an error

### ISSUE THREAD

**Issue by sartaj10** [influxdb/influxdb-python#200]
On running the following example
https://github.com/influxdb/influxdb-python/blob/master/examples/tutorial_serieshelper.py

i'm getting this error

```
Traceback (most recent call last):
  File "series_helper.py", line 45, in <module>
    MySeriesHelper(server_name='us.east-1', some_stat=155, other_stat=50)
  File "/Library/Python/2.7/site-packages/influxdb/helper.py", line 114, in __init__
    cls.commit()
  File "/Library/Python/2.7/site-packages/influxdb/helper.py", line 127, in commit
    rtn = client.write_points(cls._json_body_())
  File "/Library/Python/2.7/site-packages/influxdb/helper.py", line 146, in _json_body_
    json_point['fields'][field] = point.__dict__[field]
AttributeError: 'MySeriesHelper' object has no attribute '__dict__'
```

I'm using the version 0.9 of influxdb

**Comment by mgale** [comment]
Same problem here :(

**Comment by pegerto** [comment]
Same problem :/

**Comment by aviau** [comment]
fixed in bbe35d5e0aca8a8a9f33519ce06098c01d9b2ec5
